tests/layer_tests/pytorch_tests/pytorch_layer_test_class.py

```diff
@@ -103,6 +103,9 @@ def _test(self, model, ref_net, kind, ie_device, precision, ir_version, infer_ti
             # if None is at output we skip it
             if res_item is None:
                 continue
+            if isinstance(res_item, list):
```

Comment:
Need more comments here, we want the case (x, None, [y, None, z], None) unpack as [x, y, z]